### PR TITLE
Added code school map

### PR DESCRIPTION
**Pull request by daelencorn**
Running well on my local machine.

Moved google api url to codeschools.html.erb

array and map scripts are now seperate, both added to config/initializers/assets.rb
and both are "linked" at the bottom of codeschools.html.erb

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -15,3 +15,4 @@
 //= require bootstrap
 //= require jquery.stellar
 //= require jquery.slides.min
+//= require maps
```
Comment (emckenna) [inline app/assets/javascripts/application.js]: I think this should be "map"

**Comment by emckenna** [comment]
did you add the google mapping script ?

File: app/assets/javascripts/map.js [hunk 2]
```diff
@@ -0,0 +1,189 @@
+var locations = [
```
Comment (emckenna) [inline app/assets/javascripts/map.js]: seems like a lot of static data in this file.  Maybe we need an object here to wrap this data then feed the list with an api call.

Comment (davidmolina) [inline app/assets/javascripts/map.js]: @emckenna looks good. better to add an object for easy commits/PR later?

File: app/assets/javascripts/map.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+      google.maps.event.addListener(marker, 'click', (function(marker, i) {
+        return function() {
+          infowindow.setContent(locations[i][0]);
+          infowindow.open(map, marker);
+        }
+      })(marker, i));
+    }
+
+    google.maps.event.addDomListener(window, 'load', initialize);
```
Comment (davidmolina) [inline app/assets/javascripts/map.js]: looks like this needs a bit more work.

**Comment by davidmolina** [comment]
@daelencorn give me a few. will address in a few days. working another issue/top priority on our donations page.

**Comment by daelencorn** [comment]
@davidmolina no worries

**Comment by davidmolina** [comment]
Why can't I merge this pull request?

**Comment by emckenna** [comment]
are conflicts preventing merge?  @daelencorn pull down the lastest upstream and merge with your branch to resolve conflicts then re-push.

**Comment by davidmolina** [comment]
In meetings this morning. Will free up lunchtime.

**Comment by rickr** [comment]
Hello @daelencorn, 

Is it possible for you to rebase your changes onto master and resubmit this PR? It looks like it's pretty far out of date.

**Comment by davidmolina** [comment]
@daelencorn Possible to rebased your changes onto master and resubmit this PR?

**Pull request closed without merging**